src/components/Table/nestedCards.tsx

```diff
@@ -28,25 +28,41 @@ type Chrome = () => JSX.Element;
 type ChromeBuffer = Chrome[];
 
 export class NestedCards {
+  private columns: GridColumn<any>[];
   // A stack of the current cards we're showing
   private readonly openCards: Array<string> = [];
   // A buffer of the open/close/spacer rows we need between each content row.
   private readonly chromeBuffer: ChromeBuffer = [];
   private readonly styles: NestedCardsStyle;
 
-  constructor(private columns: GridColumn<any>[], private filteredRows: RowTuple<any>[], private style: GridStyle) {
```

Comment:
Remove the `private` keyword in the constructor as it was not clear that this was a Class variable...

Reply:
I've refactored this a bit to use the same approach for the `this.styles` since the constructor was only a single line. Let me know what you think!